resources/services/ec2/images.go

```diff
@@ -305,3 +311,28 @@ func fetchEc2ImageBlockDeviceMappings(ctx context.Context, meta schema.ClientMet
 	res <- r.BlockDeviceMappings
 	return nil
 }
+
+func resolveEc2ImageLastLaunchedTime(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	cl := meta.(*client.Client)
+	image := resource.Item.(types.Image)
+	svc := cl.Services().EC2
+	opts := ec2.DescribeImageAttributeInput{
+		Attribute: types.ImageAttributeNameLastLaunchedTime,
+		ImageId:   image.ImageId,
+	}
+	result, err := svc.DescribeImageAttribute(ctx, &opts, func(options *ec2.Options) {
+		options.Region = cl.Region
+		options.EndpointResolver = ec2.EndpointResolverFromURL(fmt.Sprintf("https://ec2.%s.amazonaws.com", cl.Region))
```

Comment:
Can we document why this is needed?

Reply:
Hm, I am not really sure, just kinda copied from the function above.